test/functional/test_runner_my.py

```diff
@@ -0,0 +1,717 @@
+#!/usr/bin/env python3
+# Copyright (c) 2014-2019 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Run regression test suite.
+
+This module calls down into individual test cases via subprocess. It will
+forward all unrecognized arguments onto the individual test scripts.
+
```

Comment:
This file `test_runner_my.py` looks like personal debug test file, should we commit it?

Reply:
I do not think so, guess it slipped in by accident.